### ISSUE TITLE
.html extension duplication in 6.1.0

### ISSUE THREAD

**Issue by ittaibaratz** [adamgruber/mochawesome-report-generator#195]
Hello there!

We have a dependency bot in our repo and in the PR that updates mochawesome-report-generator from version 6.0.1 to 6.1.0, I've noticed that the report file created has html twice in its filename. Is this an expected change?

Thank you!

```
$ marge cypress/reports/mochawesome-merged.json -o ./cypress/reports -f mochawesome.html --inline
��� Reports saved:

 /home/jenkins/workspace/exc-pueblo_PR-163/cypress/reports/mochawesome.html.html
```

**Comment by adamgruber** [comment]
Yes, the reporter no longer strips out `.html` or `.json` when used in `reportFilename`. You can simply update your command to this:
```
marge cypress/reports/mochawesome-merged.json -o ./cypress/reports -f mochawesome --inline
```

**Comment by ittaibaratz** [comment]
@adamgruber thanks for confirming. In my case its a bit tricky because our CI/CD hosts over 100 applications and I can't force them all to upgrade at the same time, but we can add some logic to strip out double HTMLs. Thanks!

**Comment by adamgruber** [comment]
So thinking on this more it's a breaking change and should have been a major version bump. I'm going to consider it a regression and put out a bug fix.

**Comment by ittaibaratz** [comment]
Awesome, thanks @adamgruber !

**Comment by adamgruber** [comment]
Fixed in mochawesome 7.1.2 (mochawesome-report-generator 6.1.1)

**Comment by ittaibaratz** [comment]
@adamgruber the dependency bot picked up 6.1.1 and reporting is back. Thanks again!
